types/lnrpc.d.ts

```diff
@@ -489,6 +489,35 @@ export interface ListPaymentsResponse {
   payments: Payment[];
 }
 
+export interface NodeUpdate {
+  addresses: string[];
+  identityKey: string;
+  globalFeatures: string;
+  alias: string;
+}
+
+export interface ChannelUpdate {
+  chanId: string;
+  chanPoint: ChannelPoint;
+  capacity: string;
+  routingPolicy: RoutingPolicy;
+  advertisingNode: string;
+  connectingNode: string;
+}
+
+export interface ClosedChannelUpdate {
+  chanId: string;
+  capacity: string;
+  closedHeight: string;
```

Comment:
Looks like this should be a number (going off the docs again).

Reply:
also returning a string